charts/plugin-bc-correios/templates/configmap.yaml

```
@@ -0,0 +1,83 @@
+  POSTGRES_HOST: {{ $component.configmap.POSTGRES_HOST | default (printf "%s-postgresql" .Release.Name) | quote }}
+  POSTGRES_PORT: {{ $component.configmap.POSTGRES_PORT | default "5432" | quote }}
+  POSTGRES_USER: {{ $component.configmap.POSTGRES_USER | default "plugin-bc-correios" | quote }}
+  POSTGRES_NAME: {{ $component.configmap.POSTGRES_NAME | default "plugin-bc-correios" | quote }}
+  POSTGRES_SSLMODE: {{ $component.configmap.POSTGRES_SSLMODE | default "disable" | quote }}
+  POSTGRES_MAX_CONNS: {{ $component.configmap.POSTGRES_MAX_CONNS | default "50" | quote }}
+  POSTGRES_MIN_CONNS: {{ $component.configmap.POSTGRES_MIN_CONNS | default "5" | quote }}
+
+  # RabbitMQ
+  # NOTE: When using external RabbitMQ, RABBITMQ_URL must be set explicitly in configmap overrides
+  # to match the actual user and host. The default URL below is only valid for bundled RabbitMQ.
+  RABBITMQ_USER: {{ $component.configmap.RABBITMQ_USER | default "plugin-bc-correios" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)